scss/partials/_footer.scss

```diff
@@ -136,6 +144,10 @@
   }
 }
 
+[class^='social-icon__link'] {
+  @include clear-decoration();
+}
+
```

Comment:
nitpick: We should probably add some sort of a marker here so that we can come back and refactor this.